framework/include/auxkernels/TagVectorArrayVariableValueAux.h

```diff
@@ -0,0 +1,29 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "TagAuxBase.h"
+#include "AuxKernel.h"
+
+/**
+ * Couple a tagged vector, and return its array value
+ */
+class TagVectorArrayVariableValueAux : public TagAuxBase<ArrayAuxKernel>
```

Comment:
we could template this further imo. like both computeValue and _v are both templatable on the return type

Reply:
It can be templatable. However, there exists the identical function for normal variable `TagVectorAux`.
So, I think we don't need to do it in this PR.